README.rst

```diff
@@ -279,6 +279,25 @@ colors`_ are supported too:
 
       * <fg=blue>Read</> the documentation at <comment>https://symfony.com/doc</>
 
+Validation
+----------
+
+When submitting a recipe, several checks are automatically executed to validate
+the recipe:
+
+* YAML files suffix must be `.yaml`, not `.yml`;
+* YAML files must be valid;
+* JSON files must be valid;
+* Aliases are only supported in the main repository, not the contrib one;
+* Aliases must not be already defined by another package;
+* The versions are valid;
+* Version defined in the directory must be the oldest amongst those defined in `version_aliases`;
+* The package must exist on Packagist;
+* The package must have at least one version on Packagist;
```

Comment:
do you mean `1 stable version`, `1 released version` (i.e. counting alpha, beta and RC versions) or are you also counting dev branches ?
If dev versions are counted, this is almost impossible to fail this validation.